nx-X11/programs/Xserver/hw/nxagent/Keystroke.c

```diff
@@ -249,6 +249,7 @@ static void parse_keystroke_file(void)
   char *homefile = "/.nx/config/keystrokes.cfg";
   char *etcfile = "/etc/nxagent/keystrokes.cfg";
 
+  /* FIXME: we do not want to have x2go references int the code */
```

Comment:
typo: s/int/in/

And I partially disagree here. We have the X2Go logo hard-coded into nxagent, too. Having hard-code X2Go related code section was a pre-requisite for merging the former x2goagent fork with the original nxagent code base.